### ISSUE TITLE
When voxes fly

### ISSUE THREAD

**Issue by Enerty** [TauCetiStation/TauCetiClassic#10098]
#### Подробное описание проблемы
Воксы делают гиперпрыжки, пролетая по несколько слоев за раз

#### Что должно было произойти
Воксы не умеют летать

#### Что произошло на самом деле
Воксы летают по космосу со скоростью фотонов

#### Как повторить
Сделать лип в космос будучи воксом

#### Дополнительная информация:
Понерфить или выпилить. Коисцин говорит что, ранее это чинилось Киборгом, по его просьбе. Если это фича, то она ничем не обоснована

**Comment by KIBORG04** [comment]
Я не помню чтобы я такое фиксил.

**Comment by UsadoSmokk** [comment]
очередное выпиливание ради выпиливание, воксы могли покидать бой, и играть как их подразумевал людук, однако пришел другой гений мысли и это уже баг

**Comment by UsadoSmokk** [comment]
![image](https://user-images.githubusercontent.com/56721822/191264371-82d9053e-dd50-413a-a9be-563a80b4dd76.png)

**Comment by flying-george** [comment]
Так-то здравый смысл, а не баг. Если ты выстрелишь с пули в космосе, она замедлится? Да и там скорость теряется после прохождения первого слоя

**Comment by Kisique** [comment]
> Так-то здравый смысл, а не баг. Если ты выстрелишь с пули в космосе, она замедлится? Да и там скорость теряется после прохождения первого слоя

скорость теряется не после слоя, а после пролета n-кол-ва тайлов. около 300.

**Comment by flying-george** [comment]
Энерти, при всем уважении, но это закон сохранения энергии. 

Ты можешь провернуть тоже самое, если оттолкнешься тростью на стуле, или стул+петушитель, но в теории сработает хуже.

**Comment by Enerty** [comment]
Это можно и из мусорной трубы исполнить и с катапульты, но это не отменяет того факта, что воксам для этого достаточно находится на твердой поверхности, опционально иметь маску для дыхания и можно покорять просторы космоса, другим подобный прикол провернуть гораздно труднее и как правило это будет сделано не для того чтобы убежать на 500 тайлов или искать дереликты

**Comment by TauKitty** [comment]
Эта проблема была упомянута в **Tau Ceti Station Forum**. Там могут быть соответствующие подробности:

https://forum.taucetistation.org/t/reklama-ishuya-i-baga/35966/3

**Comment by Helg2** [comment]
Не баг а фича, ты бы ещё катапульту пофиксил

**Comment by LudwigVonChesterfield** [comment]
Тоже имею мнение что это не баг а фича.
